### PR TITLE
add php benchmark

### LINKED ISSUE #27: Add PHP benchmark
Add a PHP grpc benchmark.

Tutorial: https://grpc.io/docs/languages/php/quickstart/

- [x] gRPC server implementation
- [x] Dockerfile to build an image with the gRPC server
- [x] update build.sh, clean.sh and bench.sh scripts

### PR DESCRIPTION
**Pull request by Trisfald**
resolves #27

### CODE REVIEW

File: php_grpc/src/HelloworldService.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Sample GRPC PHP server.
+ */
+
+use Spiral\GRPC\ContextInterface;
```
Comment (LesnyRumcajs) [inline php_grpc/src/HelloworldService.php:6]: :O

File: php_grpc/worker.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+use Spiral\RoadRunner;
+
+ini_set('display_errors', 'stderr');
+require "vendor/autoload.php";
+
+$server = new \Spiral\GRPC\Server();
+$server->registerService(\Helloworld\GreeterInterface::class, new HelloworldService());
+
+$w = new RoadRunner\Worker(new Goridge\StreamRelay(STDIN, STDOUT));
+$server->serve($w);
```
Comment (LesnyRumcajs) [inline php_grpc/worker.php:16]: the backslashes are ugly, but the implementation is still better than in `C+`. ;)

Comment (Trisfald) [reply php_grpc/worker.php:16]: true story

File: php_grpc/.rr.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+grpc:
+  listen: "tcp://:50051"
+  proto: "proto/helloworld/helloworld.proto"
+  workers:
+    command: "php worker.php"
+    pool:
+      numWorkers: 4
+
+metrics:
```
[the remark this answers is not in the document]
Comment (Trisfald) [reply php_grpc/.rr.yaml:9]: let me try to remove it

[the remark this answers is not in the document]
Comment (Trisfald) [reply php_grpc/.rr.yaml]: done

File: php_grpc/.rr.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  address: localhost:2112
+
+limit:
+  interval: 1
+  services:
+    grpc:
+      maxMemory: 200
```
[the remark this answers is not in the document]
Comment (Trisfald) [reply php_grpc/.rr.yaml]: done

**Pull request merged**